lib/matplotlib/font_manager.py

```diff
@@ -346,23 +346,6 @@ def findSystemFonts(fontpaths=None, fontext='ttf'):
     return [fname for fname in fontfiles if os.path.exists(fname)]
 
 
-def weight_as_number(weight):
```

Comment:
Same as on the other PR, please do not remove functions even if unused. Because Matplotlib is so widely used, we need to go through a deprecation cycle.